front/lib/api/assistant/global_agents.ts

```diff
@@ -184,37 +184,6 @@ function _getHelperGlobalAgent({
   };
 }
 
-function _getGPT35TurboGlobalAgent({
```

Comment:
Unfortunatel we can't remove it entirely as past conversations will require it to render properly. We need to keep that function and have this one be moved in the `RETIRED_GLOABL_AGENTS_SID`